MedscribeUI/src/components/Utilities/ThreeBtnSelector.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button } from '@mantine/core';
+import { useState, useEffect } from 'react';
+
+interface ThreeBtnSelectorProps {
+  buttonLabelOptions: string[];
+  initialSelected?: string;
+  onSelect: (value: string) => void;
```

Comment:
for consistency can we change the name of onSelect to handleSelectedPronoun

Reply:
btn selector is used for more than just pronouns so i used a generic name